xep-0234.xml

```diff
@@ -354,7 +360,12 @@
     <tr>
       <td>hash</td>
       <td>A hash of the file content, using the &lt;hash/&gt; element defined in &xep0300; and qualifed by the 'urn:xmpp:hashes:2' namespace. Multiple hashes MAY be included for hash agility.</td>
-      <td>REQUIRED when offering a file, otherwise OPTIONAL</td>
+      <td>See &lt;hash-used/&gt;</td>
+    </tr>
+    <tr>
+      <td>hash-used</td>
+      <td>Alternatively to a &lt;hash/&gt; element, the initiator can also include a &lt;hash-used/&gt; element. This comes handy to avoid reading the file twice to calculate the hash.</td>
```

Comment:
I don't think we mean "Alternatively" here because it seems that the <hash-used/> element is sent in addition to the <hash/> element (at least if I'm reading the text below correctly). An example would be helpful. English nit: "This comes in handy..." (or, less colloquially, "This avoids the need to read...").